src/electron/process_manager.ts

```diff
@@ -318,9 +318,10 @@ class ChildProcessHelper {
     };
 
     if (this.isInDebugMode) {
-      // Expose logs to the node output.  This also makes the logs available in Sentry.
-      this.process.stdout.on('data', (data) => console.log(`[STDOUT - ${processName}]: ${data}`));
-      this.process.stderr.on('data', (data) => console.error(`[STDERR - ${processName}]: ${data}`));
+      // Redirect subprocess output while bypassing the Node console.  This makes sure we don't
+      // send web traffic information to Sentry.
+      this.process.stdout!.pipe(process.stdout);
```

Comment:
Is there a way we can format the message to include the process name?
Maybe use `?` instead of `!` for safety?

Reply:
I'm not even sure we need it at all considering that in neither https://nodejs.org/api/process.html#process_process_stdout nor the typings file in @types/node are process.stdout or process.stdin marked as optional.  I took out the exclamation completely in ec4803b

Reply:
Sounds good.

Is there a way we can format the message to include the process name?